CHANGES.rst

```diff
@@ -26,6 +26,10 @@ Release History
 - Setup Nengo Bones and remote CI.
   (`#41 <https://github.com/nengo/nengo-fpga/pull/41>`__)
 
+**Changed**
```

Comment:
Should we put this under the **Fixed ** header instead maybe?

Reply:
Changed is okay. But the changelog should specify the changes were made for Nengo 3.0.0.